### PR TITLE
016/master

### PR DESCRIPTION
**Pull request by gaand**

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -8,7 +8,7 @@ def signup
     if user.valid?
       render json: user, status: :created
     else
-      head :bad_request
+      render json: user.errors, status: :bad_request
```
Comment (jrhorn424) [inline app/controllers/users_controller.rb]: 👌

File: scripts/get-examples.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```
Comment (jrhorn424) [inline scripts/get-examples.sh]: How do you feel about naming it `scripts/examples/index.sh`? I'm not asking you to change this: I'd be happy to make a PR for renaming all the scripts.

**Comment by jrhorn424** [comment]
LGTM

File: scripts/get-examples.sh (hunk 2 above)
Comment (gaand) [inline scripts/get-examples.sh]: I thought of making that change but it requires updating the README in this and all dependant repositories and so I decided to skip for now.

Comment (gaand) [inline scripts/get-examples.sh]: I think because it requires changes to other material it wait for now.

**Pull request merged**